src/Command/ReconcileCommand.php

```diff
@@ -0,0 +1,127 @@
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Webgriffe\SyliusAkeneoPlugin\ImporterInterface;
+use Webgriffe\SyliusAkeneoPlugin\ReconcilerRegistryInterface;
+use Webgriffe\SyliusAkeneoPlugin\ReconcilerInterface;
+use Webmozart\Assert\Assert;
+
+/**
+ * @psalm-suppress PropertyNotSetInConstructor $lock
+ */
+final class ReconcileCommand extends Command
+{
+    use LockableTrait;
+
+    public const SUCCESS = 0;
+
+    public const FAILURE = 1;
+
+    private const IMPORTER_OPTION_NAME = 'importer';
+
+    protected static $defaultName = 'webgriffe:akeneo:reconcile';
+
+    /**
+     * @var ReconcilerRegistryInterface
+     */
+    private $reconciliationRegistry;
+
+    /**
+     * ReconcileCommand constructor.
+     * @param ReconcilerRegistryInterface $reconciliationRegistry
+     */
+    public function __construct(ReconcilerRegistryInterface $reconciliationRegistry)
+    {
+        parent::__construct();
+        $this->reconciliationRegistry = $reconciliationRegistry;
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setDescription('Replicates the status of Akeneo products on Sylius.')
+            ->setHelp('This command allows you to reconcile the Akeneo\'s products status with the current on Sylius.')
+            ->addOption(
+                self::IMPORTER_OPTION_NAME,
+                'i',
+                InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
+                'Reconcile items only for specified importers'
+            )
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        if (!$this->lock()) {
+            $output->writeln('The command is already running in another process.');
+
+            return self::SUCCESS;
+        }
+
+        foreach ($this->getReconcilers($input) as $reconciler) {
+            $sinceDate = (new \DateTime())->setTimestamp(0);
+            $akeneoItemsToReconcile = $reconciler->getIdentifiersModifiedSince($sinceDate);
+            if (count($akeneoItemsToReconcile) === 0) {
+                $output->writeln(
+                    sprintf(
+                        'There are no <info>%s</info> entities since <info>%s</info>',
+                        $reconciler->getAkeneoEntity(),
+                        $sinceDate->format('Y-m-d H:i:s')
+                    )
+                );
+
+                continue;
+            }
+
+            $reconciler->reconcile($akeneoItemsToReconcile);
+        }
+
+        return self::SUCCESS;
+    }
+
+    /**
+     * @param InputInterface $input
+     * @return ReconcilerInterface[]
+     */
+    private function getReconcilers(InputInterface $input): array
+    {
+        $allReconcilers = $this->reconciliationRegistry->all();
+        if (count($allReconcilers) === 0) {
+            throw new \RuntimeException('There are no reconcilers in registry.');
```

Comment:
I'd not throw exception here. It's fine if there are no reconcilers.
```suggestion
            return [];
```